### PR TITLE
Handle the oldest sort parameter.

### PR DESCRIPTION
**Pull request by Natim**
See https://github.com/mozilla-services/server-syncstorage/pull/25

### CODE REVIEW

File: syncto/tests/test_functional.py [hunk 1]
```diff
@@ -244,6 +244,13 @@ def test_collection_handle_limit_and_token_parameters(self):
             "tabs", full=True, limit='2', offset='12345', sort='index',
             headers={})
 
+    def test_collection_handle_oldest_sort(self):
```
Comment (almet) [inline syncto/tests/test_functional.py]: nit: handles

File: syncto/tests/test_functional.py [hunk 2]
```diff
@@ -252,7 +259,7 @@ def test_collection_raises_with_invalid_sort_parameter(self):
         self.assertFormattedError(
             resp, 400, ERRORS.INVALID_PARAMETERS, "Invalid parameters",
             "_sort should be one of ('-last_modified', 'newest', "
-            "'-sortindex', 'index')")
+            "'-sortindex', 'index', 'last_modified', 'oldest')")
```
Comment (almet) [inline syncto/tests/test_functional.py:262]: Should this test also leave in the Sync client? (allow different sort keys in the client and not in the proxy)

Comment (Natim) [inline syncto/tests/test_functional.py:262]: I have created https://github.com/mozilla-services/syncclient/issues/19 to address this.

**Comment by leplatrem** [comment]
There is no documentation to update ?

...otherwise r+

**Comment by Natim** [comment]
Yes probably :+1:

**Pull request merged**